new-log-viewer/src/typings/worker.ts

```diff
@@ -41,6 +45,17 @@ type CursorType = {
     [T in keyof CursorArgMap]: { code: T, args: CursorArgMap[T] };
 }[keyof CursorArgMap];
 
+/**
+ * Indices for:
```

Comment:
```suggestion
 * Indices into the filtered log events collection for:
```

Reply:
I think we need some kind of name for the collection otherwise it's easy to misinterpret it as being log event indices.